### ISSUE TITLE
Add option to browse files from Folders

### ISSUE THREAD

**Issue by niptv** [beradeep/whitehole#13]
In device tab it sync all photos. means all photos in phone's storage including Whatsapp, Telegram, Screenshots etc will be visible. It's difficult to find photos which are taken by phone camera only. Photos taken by phone camera is saved inside "DCIM" folder. But there is no way to browse files by folder. Please add this feature. Your app is amazing and top quality. Thanks

**Comment by prono69** [comment]
Yes, I agree with this. All photos are clutterd in one place. It's very hard to find a specific photo.
Here are some recommendations 

- Group photos by dates like a gallary app do 
- There should be a another tab listing all folders in phone that have photos.
_I hope you understand my point. Thank you_

**Comment by prono69** [comment]
I don't think the app currently listing all the photos in a device. I can't able to see my recent photos. Instead I am seeing old photos at first 😞

**Comment by prono69** [comment]
> Yes, I agree with this. All photos are clutterd in one place. It's very hard to find a specific photo.
> Here are some recommendations 
> 
> - Group photos by dates like a gallary app do 
> - There should be a another tab listing all folders in phone that have photos.
> _I hope you understand my point. Thank you_

@beradeep any on this ?

**Comment by beradeep** [comment]
> I don't think the app correctly listing all the photos in a device. I can't able to see my recent photos. Instead I am seeing old photos at first 😞
> 
> Edit 2: So, I have 100588 photos on my phone, I downloaded a one more photo then the number is 100589 photos. **The problem is the newly downloaded photo is at the bottom of the app 💀. Am I going to scroll that much everytime? 😭😭😭**

Thanks for pointing it out. Yes, I'm aware of this. The photos aren't sorted right now. An option to sort by date needs to be implemented.

**Comment by prono69** [comment]
> > I don't think the app correctly listing all the photos in a device. I can't able to see my recent photos. Instead I am seeing old photos at first 😞
> > 
> > Edit 2: So, I have 100588 photos on my phone, I downloaded a one more photo then the number is 100589 photos. **The problem is the newly downloaded photo is at the bottom of the app 💀. Am I going to scroll that much everytime? 😭😭😭**
> 
> Thanks for pointing it out. Yes, I'm aware of this. The photos aren't sorted right now. An option to sort by date needs to be implemented.

Also it be will be great if you add something like this. (Group photos by dates like a gallary app and albums)
![Screenshot_20240808-111725567.jpg](https://github.com/user-attachments/assets/50035976-e682-4210-ba2e-be0952064de7)

**Comment by prono69** [comment]
Is this still pending?
